contracts/stake-cw20-external-rewards/src/contract.rs

```diff
@@ -2037,21 +2037,4 @@ mod tests {
         assert_pending_rewards(&mut app, &reward_addr, ADDR2, 500);
         assert_pending_rewards(&mut app, &reward_addr, ADDR3, 500);
     }
-
-    #[test]
-    fn test_math() {
-        let max_u128 = Uint256::from(Uint128::MAX);
-        let scale = Uint256::MAX.checked_div(max_u128).unwrap();
-
-        let scale_factor = Uint256::from(10u8).checked_pow(39).unwrap();
-        let min_128 = Uint128::new(1);
-        let scaled_min = Uint256::from(min_128).checked_mul(scale_factor).unwrap();
-        assert!(scaled_min > Uint256::from(Uint128::MAX));
-
-        let _max = Uint256::from(Uint128::MAX)
-            .checked_mul(scale_factor)
-            .unwrap_err();
-
-        let possible_max = Uint256::MAX.checked_div(scale_factor).unwrap();
-    }
```

Comment:
Why remove this test?

Reply:
It looks like it was just for experimentation and not actually testing the contract code